# 为您的节点 'qt_ros_node' 定义参数
# 注意: 这里的节点名需要和您在C++代码中创建节点时使用的名称一致
qt_ros_node:
  ros__parameters:

    # --- 订阅的主题名称 ---
    
    # 这个键名必须与 qt_node_constants::CIRCUIT_STATUS_TOPIC_PARAM 的值匹配
    circuit_status_topic: "circuit_status"
    
    # 这个键名必须与 qt_node_constants::REGULATOR_STATUS_TOPIC_PARAM 的值匹配
    regulator_status_topic: "voltage_regulator_status"

    # --- 发布的主题名称 ---

    # 回路命令TOPIC -- qt_node_constants::CIRCUIT_COMMAND_TOPIC_PARAM
    circuit_command_topic: "circuit_command"

    # 调压器命令TOPIC -- qt_node_constants::REGULATOR_COMMAND_TOPIC_PARAM
    regulator_command_topic: "regulator_command"

    # 清除报警TOPIC -- qt_node_constants::CLEAR_ALARM_TOPIC_PARAM
    clear_alarm_topic: "clear_alarm_command"

    # --- Service names ---
    set_system_settings_service: "set_system_settings"
    set_regulator_settings_service: "set_regulator_settings"
    set_circuit_settings_service: "set_circuit_settings"
